docs/monitoring-usage.rst

```diff
@@ -250,7 +250,7 @@ single time series. For this, you must have :mod:`pandas` installed;
 it is not a required dependency of ``google-cloud-python``.
 
 You can display CPU utilization across your GCE instances during
-the last five minutes as follows::
+the last five minutes, ending at the start of your current minute, as follows::
```

Comment:
Thanks. Looks okay, though a bit long. Sorry for editing my comment so many times. Perhaps @rimey can weigh in.